utils/defillama.ts

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+
+const baseUrl = "https://api.llama.fi/protocol/uma-protocol";
+
+interface TvlData {
+  date: string;
+  totalLiquidityUSD: number;
+}
+
+export async function getDefillamaTvl() {
+  try {
```

Comment:
we shouldnt catch these errors, wraping them in react-query will handle it for us by producing an {error} object in the hook we can check

Reply:
I agree. Updated to not catch these errors.